Dockerfile

```diff
@@ -1,12 +1,18 @@
 # CUDA 12.3
 FROM nvcr.io/nvidia/pytorch:24.02-py3
```

Comment:
they don't have a working container build for 24.07 yet. Making a note here so I don't forget to update it when they have one.